.circleci/main.yml

```diff
@@ -119,7 +119,7 @@ jobs:
     environment:
       TEST_RESULTS: /tmp/test-results
       GOPROXY: https://proxy.golang.org
-    parallelism: 8
+    parallelism: 10
```

Comment:
I think @djanicekpach has a PR up for this. one you might want to revert ?

Reply:
This is the latest committed version (I merged master into this branch after making my own changes).